main/getting-started/intro-zoe.md

```diff
@@ -0,0 +1,279 @@
+
+
+## Installing and instantiating contracts on Zoe
+
+
+To use a contract, it needs to be *installed* on Zoe. This is putting the contract’s source code, consisting of JavaScript plus ERTP and Zoe API requests, on Zoe. 
+
+
+When a contract is installed on Zoe, the operation returns an `installationHandle`, which is an *opaque JavaScript object*  used to refer to the installed contract source code. Agoric uses such objects as handles since they cannot be forged. For example
+```js
+const installationHandle = zoe.install(sourceCode, moduleFormat);
+```
+
+
+A contract has to be *instantiated* for it to actually be used. A contract can be instantiated by calling `E(zoe).makeInstance(installationHandle).` Multiple *instances* of any Zoe-installed contract can execute simultaneously. For clarity, in this document a *contract* is the installed code that defines how the contract works. A *contract instance* is an active execution of the contract code. 
+
+
+For example, let’s say a realtor has a standard agreement they use when selling a client’s house. The contract would be the code that defines how that standard agreement works. Every time the realtor has a new house to sell, they instantiate a new instance of their standard contract to use for that specific property. If they have ten houses up for sale, they have ten different contract instances.
+
+
+In addition to an `installationHandle`, when the contract is instantiated, Zoe needs to know what kind of ERTP assets the contract will work with. In Atomic Swap’s test code, the full instantiation call is:
+
+
+```js
+const issuerKeywordRecord = harden({
+  Asset: moolaIssuer,
+  Price: simoleanIssuer,
+});
+const aliceInvite = await E(zoe).makeInstance(
+  installationHandle,
+  issuerKeywordRecord, 
+});
+```
+Let’s work backwards from `makeInstance()`’s arguments. As you’ll recall from the [Introduction to ERTP](https://agoric.com/documentation/getting-started/ertp-introduction.html#issuers-and-mints), *issuers* map minted assets to purses and payments. The *keyword record* is made up of two key:value pairs. The key names must be ASCII and capitalized; they are the same as are used in the contract code, in this case `Price` and `Asset`. `Price` is for what is wanted, and `Asset` is for what is being sold. `Price` is for what will be swapped for it (from the perspective of the user making the instance; it’d be the opposite for someone who wanted what Alice is offering). The price is denominated in simoleons, so that keyword needs the simoleon Issuer associated with it. The asset is denominated in moola, so that keyword needs the moola Issuer as its value. 
```

Comment:
> `Price` is for what is wanted, and `Asset` is for what is being sold. `Price` is for what will be swapped for it 

`Asset` is for what is being sold. `Price` is for what will be swapped for it 

I'd drop the first clause. The second description of Price is clearer and sufficient